canonical-webteam/global-nav
# Refactor the code base

## Done
Updated the classes used to reflect BEM. Refactored the Sass to use an ITCSS structure.

## QA
- Pull this branch
- Run `make develop`
- Then open `index.html` in your browser
- Check the more menu opens

_No small screen styling yet. That is a follow-up_

## Details
Card: https://trello.com/c/BPDeQY7Z
